packages/@coorpacademy-components/src/molecule/card/index.js

```diff
@@ -101,6 +107,7 @@ CardBackground.contextTypes = {
 };
 
 CardBackground.propTypes = {
+  theme: PropTypes.oneOf(keys(THEMES)),
```

Comment:
```suggestion
  theme: PropTypes.oneOf(keys(THEMES)),
```

le root a deja le theme coorpamanager